### PR TITLE
Matt simple

### PR DESCRIPTION
**Pull request by Canthv0**
**Issue:**
Script failed to run after downloading.
Was generating large numbers of null exception errors even after correcting basic script failure.

**Reason:**
Correct errors to bring script back to function.

**Fix:**
Fixed issue with Parameters.
Added null check for modified properties.

**Validation:**
Ran script against customer provided dataset and received correct output.

### CODE REVIEW

**Review by bill-long — CHANGES_REQUESTED** [review]
```
RuleName                             Severity ScriptName                   Line Message
--------                             -------- ----------                   ---- -------
PSUseDeclaredVarsMoreThanAssignments Warning  Get-SimpleAuditLogReport.ps1 68   The variable 'ResultSet' is assigned
                                                                                but never used.
```

Please run .build\CodeFormatter and fix any reported issues until clean.

File: Admin/Get-SimpleAuditLogReport.ps1 [hunk 1]
```diff
@@ -193,10 +191,7 @@ Process {
...
     # Add the object to the array to be returned
-    $ResultSet = $ResultSet + $Result
+    $Result
 }
 # Final steps
-End {
-    # Return the array set
-    Return $ResultSet
-}
+End {}
```
Comment (dpaulson45) [inline Admin/Get-SimpleAuditLogReport.ps1:197]: should remove this if it is empty.

**Review by dpaulson45 — CHANGES_REQUESTED** [review]

File: Admin/Get-SimpleAuditLogReport.ps1 (hunk 1 above)
Comment (Canthv0) [reply Admin/Get-SimpleAuditLogReport.ps1]: <shrug> was left there for consistency with the Begin / Process / End methodology but doesn't matter either way so took it out.

**Comment by Canthv0** [comment]
Everything fixed should be good to go.

**Review by bill-long — APPROVED** [review]

File: Admin/Get-SimpleAuditLogReport.ps1 (hunk 1 above)
Comment (dpaulson45) [reply Admin/Get-SimpleAuditLogReport.ps1]: True, but there wasn't even a Process section in the script. Which i am surprised that we don't get a rule error on this being that we set the one parameter to be accepted from the pipeline.

**Review by dpaulson45 — APPROVED** [review]

**Pull request merged**